commands/displayers/activations.go

```diff
@@ -0,0 +1,120 @@
+package displayers
+
+import (
+	"fmt"
+	"io"
+	"strings"
+	"time"
+
+	"github.com/apache/openwhisk-client-go/whisk"
+)
+
+type Activation struct {
+	Activations []whisk.Activation
+}
+
+var _ Displayable = &Activation{}
+
+// ColMap implements Displayable
+func (a *Activation) ColMap() map[string]string {
+	return map[string]string{
+		"Datetime":     "Datetime",
+		"Status":       "Status",
+		"Kind":         "Kind",
+		"Version":      "Version",
+		"ActivationId": "Activation ID",
```

Comment:
```suggestion
		"ActivationID": "Activation ID",
```

For consistency, we generally prefer `ID` over `Id`.